#!/usr/bin/env python

PACKAGE = "mur_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# For X axis
gen.add("p_x", double_t,   0, "Proportional parameter X",       58, 0, 100)
gen.add("i_x", double_t,   0, "Integral parameter X",           25, 0, 100)
gen.add("d_x", double_t,   0, "Derivative parameter X",         32, 0, 100)

gen.add("pos_x", double_t,  0, "Position X desire",     0, -100, 100)

# For Y axis
gen.add("p_y", double_t,   0, "Proportional parameter Y",       58, 0, 100)
gen.add("i_y", double_t,   0, "Integral parameter Y",           25, 0, 100)
gen.add("d_y", double_t,   0, "Derivative parameter Y",         32, 0, 100)

gen.add("pos_y", double_t,  0, "Position Y desire",     0, -100, 100)


exit(gen.generate(PACKAGE, "mur_position_control", "MurPositionControl"))
